File: src/main/scala/com/kirkwoodwest/scalawoods/utils/RegexUtil.scala
package com.kirkwoodwest.scalawoods.utils

import java.util.regex.Matcher
import java.util.regex.Pattern

object RegexUtil {
  def stringMatch(lookup_string: String, `match`: String): Boolean = {
    val p = Pattern.compile("" + `match`, Pattern.CASE_INSENSITIVE)
    val m = p.matcher(lookup_string)
    val b = m.find
    b
  }
}

File: src/main/java/com/kirkwoodwest/openwoods/utils/RegexUtil.java
package com.kirkwoodwest.openwoods.utils;

import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class RegexUtil {
	static public boolean stringMatch(String lookup_string, String match){
		Pattern p = Pattern.compile("" + match, Pattern.CASE_INSENSITIVE);
		Matcher m = p.matcher(lookup_string);
		boolean b = m.find();
		return b;
	}
}
